src/actions/BitPayActions.js

```diff
@@ -117,30 +147,38 @@ export async function launchBitPay(dispatch: Dispatch, uri: string, currencyWall
   const instructionOutput = invoiceInstruction.outputs[0]
 
   // Make the spend to generate the tx hexes
+  const requiredFeeRate = invoiceInstruction.requiredFeeRate
   const spendInfo: EdgeSpendInfo = {
     currencyCode,
-    customNetworkFee: { satsPerByte: invoiceInstruction.requiredFeeRate },
     spendTargets: [
       {
         nativeAmount: instructionOutput.amount.toString(),
         publicAddress: instructionOutput.address
       }
-    ]
+    ],
+    otherParams: {
+      paymentProtocolInfo: {
+        merchant: {
+          requiredFeeRate
+        }
+      }
+    }
   }
-  const wallet = currencyWallets[walletId]
-  const unsignedTx = await wallet.makeSpend(spendInfo)
+  // $FlowFixMe
```

Comment:
Flow doesn't like optional chaining when calling methods but checking for null beforehand should be enough. Line 118 should be sufficient to satisfy flow and you can remove the ?s (but we don't like sketchy nulls so it should be if (selectedWallet == null) return